docs/source/algorithms/JoinRuns-v1.rst

```diff
@@ -0,0 +1,124 @@
+
+.. testcode:: JoinRunsExample
+   
+    # Create input workspaces
+    list = []
+    for i in range(3):
+        ws = "ws_{0}".format(i)
+        CreateSampleWorkspace(Function="One Peak", NumBanks=1, BankPixelWidth=2,
+                              XMin=i*100, XMax=(i+1)*100, BinWidth=50,
+                              Random=True, OutputWorkspace=ws)
+        ConvertToPointData(InputWorkspace=ws, OutputWorkspace=ws)
+        list.append(ws)
+
+    # Join the workspaces
+    out = JoinRuns(list)
+
+    # Check the output
+    print("out has {0} bins with x-axis as: {1}, {2}, {3}, {4}, {5}, {6}".
+          format(out.blocksize(), out.readX(0)[0], out.readX(0)[1], out.readX(0)[2],
+                 out.readX(0)[3], out.readX(0)[4], out.readX(0)[5]))
+
+Output:
+
+.. testoutput:: JoinRunsExample
+
+    out has 6 bins with x-axis as: 25.0, 75.0, 125.0, 175.0, 225.0, 275.0
+
+**Example - JoinRuns with a numeric log**
+
+.. testcode:: JoinRunsLogExample
+
+    # Create input workspaces
+    list = []
+    for i in range(3):
+        ws = "ws_{0}".format(i)
+        CreateSampleWorkspace(Function="One Peak", NumBanks=1, BankPixelWidth=2,
+                              XMin=i*100, XMax=(i+1)*100, BinWidth=100,
+                              Random=True, OutputWorkspace=ws)
+        ConvertToPointData(InputWorkspace=ws, OutputWorkspace=ws)
+        AddSampleLog(ws, LogName='LOG',LogType='Number', LogText=str(5*i))
+        list.append(ws)
+
+    # Join the workspaces
+    out = JoinRuns(list, SampleLogAsXAxis='LOG')
+
+    # Check the output
+    print("out has {0} bins with x-axis as: {1}, {2}, {3}".
+          format(out.blocksize(), out.readX(0)[0], out.readX(0)[1], out.readX(0)[2]))
+
+Output:
+
+.. testoutput:: JoinRunsLogExample
+
+    out has 3 bins with x-axis as: 0.0, 5.0, 10.0
+
+**Example - JoinRuns with a numeric time series log**
+
+.. testcode:: JoinRunsTSLogExample
+
+    import datetime
+    # Create input workspaces
+    list = []
+    for i in range(3):
+        ws = "ws_{0}".format(i)
+        CreateSampleWorkspace(Function="One Peak", NumBanks=1, BankPixelWidth=2,
+                              XMin=i*100, XMax=(i+1)*100, BinWidth=50,
+                              Random=True, OutputWorkspace=ws)
+        ConvertToPointData(InputWorkspace=ws, OutputWorkspace=ws)
+
+        for j in range(2):
+            AddTimeSeriesLog(ws, Name='LOG',Time=str(datetime.datetime.now()), Value=str(10*i+0.25*j))
+
+        list.append(ws)
+
+    # Join the workspaces
+    out = JoinRuns(list, SampleLogAsXAxis='LOG')
+
+    # Check the output
+    print("out has {0} bins with x-axis as: {1}, {2}, {3}, {4}, {5}, {6}".
+          format(out.blocksize(), out.readX(0)[0], out.readX(0)[1], out.readX(0)[2],
+          out.readX(0)[3], out.readX(0)[4], out.readX(0)[5]))
+
+Output:
+
+.. testoutput:: JoinRunsTSLogExample
+
+    out has 6 bins with x-axis as: 0.0, 0.25, 10.0, 10.25, 20.0, 20.25
+
+
+.. categories::
+
+.. sourcelink::
+
```

Comment:
I think it is worth adding one example which shows the sample log behaviour. Then as the docs link to `MergeRuns` users can find more information in the docs there if they need it.